### ISSUE TITLE
Should support undici timeout err.code

### ISSUE THREAD

**Issue by yohay-ma** [fastify/fast-proxy#33]
## 🐛 Bug Report

undici timeout err.code=UND_ERR_REQUEST_TIMEOUT is not handled properly, therefore instead of getting error 504 I get error 500

We need to add in index.js line 88 the undici timeout error code as well:

`} else if (err.code === 'ECONNRESET' || err.code === 'UND_ERR_REQUEST_TIMEOUT' ) {
`
## To Reproduce

in undici set the time out to a low number:
```
    undiciOptions: {
      connections: 4000,
      pipelining: 20,
      proxyCacheSize: 50,
      proxyCacheTtl: 1000 * 60 * 60 * 24,
      requestTimeout: 1000
    }
```
in test server open a route with sleep:
for example:
```js
  .get('/api/v1/my-route/timeout', async (req, res) => {
    await sleep(3000);
    res.end();
  })
```

## Expected behavior

Should handle timeout code from undici

Note: I think the following error codes should also be added: `ESOCKETTIMEDOUT` and `ETIMEDOUT`

## Your Environment

- *node version*: 12
- *fast-proxy version*: 1.6.3
- *os*: Mac

**Comment by yohayg** [comment]
See PR: https://github.com/fastify/fast-proxy/pull/34

**Comment by yohayg** [comment]
@jkyberneees thanks for merging both items.
When are you going to bump a version so I will be able to consume it from npmjs?

**Comment by jkyberneees** [comment]
Hi @yohayg could you please check: https://github.com/fastify/fast-proxy/pull/38, I guess this is blocking the release. 
I would appreciate if you can confirm that this issue appears on your side as well.

**Comment by jkyberneees** [comment]
Feature is available in v1.7.0 release.
